fanctrl.py

```diff
@@ -463,20 +486,36 @@ class FanController:
     tempHistory = collections.deque([0] * 100, maxlen=100)
     active = True
     timecount = 0
+    noBatteryMode = False
```

Comment:
Perhaps the "noBatteryMode" should be in the hardware controller instead, as it's only purpose is to interface with the hardware to parse temperature and fan speed?